packages/E2ETest/windows/ReactUWPTestApp/App.xaml.cs

```diff
@@ -54,6 +58,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
         {
             base.OnLaunched(e);
             SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = AppViewBackButtonVisibility.Collapsed;
+            ApplicationView.GetForCurrentView().TryResizeView(new Size(1024, 768));
```

Comment:
You had mentioned issues having consistent layout sizes. Did this work?

Reply:
Yes, by default we will launch the app at 800x600 and I also added the code to fail and don't do comparison if size is not 800x600 in tree dump too.

---
In reply to: [357847530](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r357847530) [](ancestors = 357847530)